vlasovsolver/cpu_trans_map_amr.hpp

```diff
@@ -208,7 +207,17 @@ void prepareSeedIdsAndPencils(const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Ge
 // pencils used for AMR translation
 static std::array<setOfPencils,3> DimensionPencils;
 
+// Only communicate required cells
 void flagSpatialCellsForAmrCommunication(const dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpiGrid,
                                          const std::vector<CellID>& localPropagatedCells);
 
+// declare sets used for storing cells participating in all-local translation
+// defined in cpu_trans_map_amr.cpp
+extern std::unordered_set<CellID> LocalSet_x;
+extern std::unordered_set<CellID> LocalSet_y;
+extern std::unordered_set<CellID> LocalSet_z;
```

Comment:
They are only used in that cpp file, why do they need an extra extern declaration here?